code/game/objects/items/kitchen.dm

```diff
@@ -150,8 +150,8 @@
 	item_state = "knife"
 	desc = "A military combat utility survival knife."
 	embedding = list("pain_mult" = 4, "embed_chance" = 65, "fall_chance" = 10, "ignore_throwspeed_threshold" = TRUE)
-	force = 20
-	throwforce = 20
+	force = 16
+	throwforce = 16
```

Comment:
```suggestion
	throwforce = 20
```
if you chuck it at someone the unique embedding stats (specifically, "ignore_throwspeed_threshold") means it's still gonna have a good chance of lodging into someone (or the wall next to them if you fuck it up)

Reply:
and even if it doesn't lodge into them you're still essentially giving them your knife

Reply:
Yeah that's fair, throwforce damage isn't something that's really consistent.